datafusion-cli/tests/sql/integration/glob_test.sql

```diff
@@ -0,0 +1,15 @@
+-- Test glob function with files available in CI
+-- Test 1: Single CSV file - verify basic functionality
+SELECT COUNT(*) AS cars_count FROM glob('../datafusion/core/tests/data/cars.csv');
+
+-- Test 2: Data aggregation from CSV file - verify actual data reading
+SELECT car, COUNT(*) as count FROM glob('../datafusion/core/tests/data/cars.csv') GROUP BY car ORDER BY car;
```

Comment:
I think another usecase that @robtandy  had was "a list of multiple files" -- like is there some way to select exactly two files? Something like

```sql
glob('[../datafusion/core/tests/data/cars.csv',  '../datafusion/core/tests/data/trucks.csv', ])
```

Perhaps 🤔